src/TableV8/styled.tsx

```diff
@@ -21,93 +21,130 @@ import clsx from 'clsx';
   BORDER: colors.grey_2,
-  ROW_HIGHLIGHT_BACKGROUND: colors.grey_3,
+  BACKGROUND_HIGHLIGHT: colors.grey_3,
 };
 
-const TableContainer = (props: React.PropsWithChildren<{ className?: string }>) => (
+const TableContainerComp = (props: React.PropsWithChildren<{ className?: string }>) => (
   <div {...props} className={clsx(TABLE_CLASSES.TABLE_CONTAINER, props.className)} />
 );
-export const StyledTableContainer = styled(TableContainer)`
+export const TableContainer = styled(TableContainerComp)`
   width: 100%;
   max-width: 100%;
   overflow-x: auto;
+  position: relative;
+  box-sizing: border-box;
 `;
 
-const Table = (
+const TableComp = (
   props: React.PropsWithChildren<{
     className?: string;
     withSideBorders?: boolean;
   }>,
 ) => <table {...props} className={clsx(TABLE_CLASSES.TABLE, props.className)} />;
-export const StyledTable = styled(Table, {
-  shouldForwardProp: (prop) => isPropValid(prop) && !['withSideBorders'].includes(prop),
+export const TableStyled = styled(TableComp, {
+  // naming conflict with <Table />
+  shouldForwardProp: (prop) => isPropValid(prop),
 })`
   border: solid 1px ${COLORS.BORDER};
   border-right-width: ${(props) => `${props.withSideBorders ? '1' : '0'}px`};
   border-left-width: ${(props) => `${props.withSideBorders ? '1' : '0'}px`};
   border-collapse: collapse;
   width: 100%;
-  border-spacing: 0;
   .${TABLE_CLASSES.TH}, .${TABLE_CLASSES.TD} {
-    display: flex;
-    flex: 100 1 auto;
-    width: 100px;
+    padding: 0;
+    &:not(:last-of-type) {
+      border-right: 1px solid ${COLORS.BORDER};
+    }
+  }
+  .${TABLE_CLASSES.TH_WRAPPER}, .${TABLE_CLASSES.TD_WRAPPER} {
     padding: 2px 8px;
     font-family: Work Sans, sans-serif;
     font-size: 12px;
     line-height: 1.33;
-    min-height: 24px;
+    min-height: 28px;
+    text-align: left;
+    display: flex;
     align-items: center;
-    &:not(:last-of-type) {
-      border-right: 1px solid ${COLORS.BORDER};
-    }
+    box-sizing: border-box;
+  }
+  .${TABLE_CLASSES.TH_WRAPPER}, .${TABLE_CLASSES.SORT_BUTTON} {
+    font-weight: bold;
   }
 `;
 
-const TableHead = (props: React.PropsWithChildren<{ className?: string }>) => (
+const TableHeadComp = (props: React.PropsWithChildren<{ className?: string }>) => (
   <thead {...props} className={clsx(TABLE_CLASSES.THEAD, props.className)} />
 );
-export const StyledTableHead = styled(TableHead)``;
+export const TableHead = styled(TableHeadComp)``;
 
-const TableHeader = (props: React.PropsWithChildren<{ className?: string; colSpan?: number }>) => (
-  <th {...props} className={clsx(TABLE_CLASSES.TH, props.className)} />
-);
-export const StyledTableHeader = styled(TableHeader)`
-  text-align: left;
+// the TH element should only for table structure.
+// use TableHeaderWrapper for additional styling within the TH.
+const TableHeaderComp = (
+  props: React.PropsWithChildren<{
+    canSort?: boolean;
+    className?: string;
+    colSpan?: number;
+    sorted?: 'asc' | 'desc' | false;
```

Comment:
the usage implies there are only two valid values `asc` and `falsy` 
it's optional so explicitly passing `false` is unlikely, and `desc` doesn't appear to be used

Reply:
unsorted columns pass the value false, but in tables without sorting they don't pass anything